students/JaeKim/lesson03/assignment/basic_operations.py

```diff
@@ -0,0 +1,108 @@
+import logging
+from customers_model import Customer, TestCustomer
+import customers_model as cm
+
+
+def add_customer(customer_id, name, lastname, home_address, phone_number, email_address, status, credit_limit,
+                 stage='prod'):
+    """
+    :param customer_id:
+    :param name:
+    :param lastname:
+    :param home_address:
+    :param phone_number:
+    :param email_address:
+    :param status:
+    :param credit_limit:
+    :param stage:
+    :return:
+    """
+    try:
+        credit_limit = float(credit_limit)
+    except ValueError as err:
+        logging.error(err)
+        raise
+
+    args = {
+        'customer_id' : customer_id,
+        'first_name' : name,
+        'last_name' : lastname,
+        'home_address' : home_address,
+        'phone_number' : phone_number,
+        'email_address' : email_address,
+        'status' : status,
+        'credit_limit' : credit_limit
+    }
+
+    if stage == 'prod':
+        new_customer = Customer.create(**args)
+        new_customer.save()
+
+    if stage == 'dev':
+        new_customer = TestCustomer.create(**args)
+        new_customer.save()
+
+
+def search_customer(customer_id, stage='prod'):
+    """
+    :param customer_id:
+    :param stage:
+    :return:
+    """
+    try:
+        if stage == 'prod':
+            results = Customer.get(cm.customer_id == customer_id)
+
+        if stage == 'dev':
+            results = TestCustomer.get(TestCustomer.customer_id == customer_id)
+    except Exception:
+        return False
```

Comment:
You should log your exceptions, especially because you are catching all of them. An exception that you aren't expecting could happen, and you would never know.